redisvl/index.py

```diff
@@ -510,14 +520,26 @@ async def create(self, overwrite: Optional[bool] = False):
         Raises:
             redis.exceptions.ResponseError: If the index already exists.
         """
-        exists = await self.exists()
-        if exists and overwrite:
+        # TODO - enable async version of this
+        # check_redis_modules_exist(self._redis_conn)
+
+        if not self._fields:
```

Comment:
should this be allowed?

Reply:
This is the current behavior of the synchro version of the `create` method, and was just missing here